src/components/loadJSON.js

```diff
@@ -5,19 +5,21 @@ var maxInFlight = 10;
 
 module.exports.create = function create_json_parse_stream(dataDirectory) {
   return parallelStream(maxInFlight, function(record, enc, next) {
-    fs.readFile(dataDirectory + record.path, function(err, data) {
+    var full_file_path = dataDirectory + record.path;
+    fs.readFile(full_file_path, function(err, data) {
       if (err) {
+        console.error('exception reading file ' + full_file_path);
         next(err);
       } else {
         try {
           var object = JSON.parse(data);
           next(null, object);
         } catch (parse_err) {
-          console.error('exception on %s:', record.path, parse_err);
+          console.error('exception parsing JSON in file %s:', record.path, parse_err);
           console.error('Inability to parse JSON usually means that WOF has been cloned ' +
                         'without using git-lfs, please see instructions here: ' +
                         'https://github.com/whosonfirst/whosonfirst-data#git-and-large-files');
-          next(null, {});
+          next(parse_err);
         }
```

Comment:
No, lint updates are fine to change. But the changes to this file change the behavior, because any error will now stop the stream. We want to do that (As mentioned in [this comment](https://github.com/pelias/pelias/issues/477#issuecomment-265002768)), but it seems better to make that change in its own PR.